CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+## [v4.0.0](https://github.com/symfony/webpack-encore/releases/tag/v4.0.0)
+
+This major release makes Encore compatible with [Yarn Plug'n'Play](https://yarnpkg.com/features/pnp) and [pnpm](https://pnpm.io/fr/).
```

Comment:
Take the `/fr` off of the pnpm URL. It's also included 3x in the `README.md` of 3 of the "test apps".